modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1193 @@
+      tooltip: '<sidebar-name>',
+      icon: '<icon-name>',
+      onSetup: (api) => {
+        console.log('Render panel', api.element());
+      },
+      onShow: (api) => {
+        console.log('Show panel', api.element());
+      },
+      onHide: (api) => {
+        console.log('Hide panel', api.element());
+      }
+    });
+  }
+});
+----
+
+NOTE: The sidebar presents with no animation or other effect. It is just there, along with any other initialised {productname} editor components. Also, as noted above, the sidebar does not take focus, which remains with the main editor.
+
+////
+A common envisioned use is to initialise a {productname} editor with the Premium Plugin Comments sidebar automatically present.
+
+After adding the Comments Premium Plugin, to add a Comments sidebar, add `sidebar_show: 'showcomments'` to `tinymce.init`:
+
+[source,js]
+----
+tinymce.init({
+  selector: "textarea", // change this value according to your HTML
+  sidebar_show: 'showcomments',
+});
+----
+////
+
+
+=== New `newline_behavior` option controls what happens when the Return or Enter key is pressed or the `mceInsertNewLine` command is used
+
+Setting `forced_root_block` to `false` or an empty string was deprecated in {productname} 6.0. This option now only accepts valid block elements.
+
+However, some use cases relied on `forced_root_block: false` to keep material created in {productname} for use and presentation elsewhere from being placed inside the `<p>` block element.
+
+The `newline_behavior` option has been added to {productname} 6.1 to cater to these use-cases.
+
+`newline_behavior` affects what is added to a document when the `mceInsertNewLine` command is run or when any of the four _create a new line_ keystrokes are pressed: *Return*, *Enter*, *Shift-Return*, or *Shift-Enter*.
+
+By default these keystrokes work as follows:
+
+1. pressing *Return* or *Enter* creates a new paragraph.
++
+For example, in ordinary text, pressing *Return* closes the current `<p>` tag, moves the insertion point in the {productname} editor to a new line and opens a new `<p>` tag.
+   
+2. pressing *Shift+Return* or *Shift+Enter* creates a new line.
++
+For example, in ordinary text, pressing *Shift+Return* enters two `<br>` tags and moves the insertion point in the {productname} editor to a new line.
+
+The `newline_behavior` option takes one of four values:
+
+* `block`
++
+`block` sets {productname} to treat all four keystrokes as if they are *Return*.
+  
+* `linebreak`
++
+`linebreak` sets {productname} to treat all four keystrokes as if they are *Shift+Return*.
+  
+* `invert`
++
+`invert` swaps the behavior or *Return* and *Shift+Return* (and *Enter* and *Shift+Enter*). That is, pressing *Return* results in *Shift-Return* behavior, and vice versa.
+
+* `default`
++
+`default`, as its name suggests, sets {productname} to do what it does by default when *Return*, *Enter*, *Shift+Return*, or *Shift+Enter* are pressed.
```

Comment:
This does not render very well... the content is also confusing me as it just looks like repetition. Maybe a description list might be better? Otherwise maybe we should just avoid duplicating like `block - sets TinyMCE to treat all four keystrokes as if they are a Return`.

![image](https://user-images.githubusercontent.com/1575550/178645730-ffe33e56-8494-4635-807f-73efa1343466.png)

Reply:
How about a simple table instead?

Reply:
NB: I’ve already pushed an update that puts this into a simple table.